sql/moz-fx-data-shared-prod/stripe_external/itemized_payout_reconciliation_v5/metadata.yaml

```diff
@@ -17,5 +17,5 @@ scheduling:
     - --time-partitioning-field=automatic_payout_effective_at
   # report data isn't generally ready when the dag starts, so retry for a while
   retry_delay: 30m
-  retries: 10
+  retries: 32
```

Comment:
Since we don't yet know when the report becomes available, to avoid possibly having to do this again I'd consider increasing this to 47 to cover the entire day.